README.md

```diff
@@ -40,10 +37,9 @@ cargo test
 ```
 
 ## :scroll: Documentation
-The following documents are relevant to this service:
-- [Concept of Operations](TODO)
-- [Requirements & User Stories](TODO)
-- [SDD](./docs/sdd.md)
+The following documents are relevant to this library:
+- [Concept of Operations](FIXME)
+- [Requirements & User Stories](FIXME)
```

Comment:
Is there a good reason for excluding the SDD from this?

Reply:
Will re-add.

Momentarily pondered if libraries should have SDDs, I now believe they should.

Reply:
Fixed: https://github.com/Arrow-air/template-rust-lib/pull/1/commits/10f99ba487d694fce868138e25b71987138371f9